### PR TITLE
Code Review for [m6-create-views] | Dev2Dev Blog Rails Project

### PR DESCRIPTION
**Pull request by luigirazum**
# Hello Dear Code Reviewers Team 👋,
### - 🎯 General requirements
- [X] Make sure that there are _**no** linter errors_.
  - 🟢 `rubocop` ran and **no offenses** were detected.
- [X] Make sure that you used _**correct** flow_ `git-flow`.
  - 🟢 The <kbd>Milestone</kbd> branch `m6-create-views` was created from `dev` to follow the `git-flow`.
- [X] Make sure that you documented your work _in a **professional way**_.
  - 🟢 All the information about the project was added to the `README.md` file, previously.

### - 🎯 Project requirements
- Implement the **design** from the sneak peek **wireframes**.
  - [X] **Views** for `index` and `show` actions were created for both `Users` and `Posts` `Controllers`.

- **Analyze** the **wireframes** deeply and try to _identify_ all the _necessary elements_ and **how** a user would **navigate** from one page to another.
  - [X] Several **partials** files were created after the analysis of the wireframes.
  - [X] Some partials are **shared** among the views.
  - [X] A simple and dynamic `navbar` was added to the main `Layout` so the user can return from the **Full Post** page to the **User** page.
  - [X] A link was added to the `Post.title` so the user can go to the **Full Post** page.
  - [X] A `See all posts` link was added to the `posts#index` view, so if the user has no `posts` then it is not showed.
    
- Use methods that you have **created** with the `Models`.
  - [X] `most_recent_post` and `most_recent_comments` methods were used in the views.
  - [X] `posts_counter` and `likes_counter` were used as well.

### Luis 🙋‍♂️

### CODE REVIEW

File: app/views/posts/_post.html.erb [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+</h3>
+<h2><%= post.title %></h2>
+<p>
+  <%= post.text %>
+</p>
+<section>
+  <h3> <%= post.comments.size.positive? ? 'Comments' : 'No comments yet!' %></h3>
+  <% post.comments.each do |comment| %>
+    <%= render partial: 'shared/comment', locals: { comment: comment } %>
+  <% end %>
```
Comment (elisha2kyakpo1) [inline app/views/posts/_post.html.erb:22]: - [ ] You did a great job working on this project, All the major functionalities were done according to the project requirements, but the wireframe design was not followed. Kindly make your app page designs look the same as the wireframes. This means that you will need to add a bit of CSS to the pages.

yours


design

**Review by elisha2kyakpo1 — CHANGES_REQUESTED** [review]
Hi 👋 @luigirazum, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️ The PR title and description are good and professional.
✔️ No linter errors.
👍 Descriptive readme file.
✔️ Model methods used correctly👏
👏 The app works as expected.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by nedjwab — APPROVED** [review]
# 🟢 Project Approved! 🏆

## Hi  @luigirazum                    👋

Your project is complete! ✔️✔  🏁

Your code was very professional 💯

There is nothing else to say other than... _it's time to merge it_ :shipit:
Congratulations! 🎉

 
## Highlights 🌟

- ✔️ Descriptive commmit messsages
- ✔️ Descriptive PR description
- ✔️ No linter Errors


- The code is top-notch! 💯 and have a huge potential.

Cheers and Happy coding!👏👏👏

## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.


_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by luigirazum** [comment]
Excuse me @nedjwab, what are the required changes?
Previous reviewer asked for styling.
What is the issue now?

**Pull request merged**